.circleci/heroku_deploy.sh

```diff
@@ -2,9 +2,11 @@
 
 if git remote | grep heroku > /dev/null; then
   git fetch heroku
-  git push --force heroku $CIRCLE_SHA1:master
+  DEPLOY_ENV=staging yarn deploy
+  git push git@github.com:artsy/force.git $CIRCLE_SHA1:staging --force
```

Comment:
We deploy to prod by opening a PR from `staging` to `release`, so we still want to push after a successful heroku deploy.